liquibase-standard/src/main/java/liquibase/logging/core/JavaLogService.java

```diff
@@ -30,13 +37,32 @@ public Logger getLog(Class clazz) {
                 utilLogger.setParent(parent);
             }
             logger = new JavaLogger(utilLogger);
+            setChannelLogLevel(getLogName(clazz), utilLogger);
 
             this.loggers.put(clazz, logger);
         }
 
         return logger;
     }
 
+    private void setChannelLogLevel(String channelName, java.util.logging.Logger logger) {
```

Comment:
It looks like this is the same code that exists in `LiquibaseCommandLine#configureLogging`. Can this method be used there too?

Also can you update the pr description and explain what was happening and how this fixes it?

Reply:
It's based off of that, but it differs some.